packages/e2e-tests/cypress/support/eligibility/granted-or-refused-application/select-planning-application-decision.js

```diff
@@ -1,14 +1,16 @@
-import {getGrantedRadio, getRefusedRadio, getNoDecisionReceivedRadio, getPlanningApplicationDecisionError} from '../page-objects/granted-or-refused-application-po'
+import {getGrantedRadio, getRefusedRadio, getNoDecisionReceivedRadio
+} from '../page-objects/granted-or-refused-application-po'
 
 export const selectPlanningApplicationDecision = (decision)=>{
+  console.log('aaaaaaa '+decision)
     switch(decision){
         case 'Granted':
```

Comment:
Might be a better idea to use constants

Reply:
This is Shilpa's domain so going to leave as is. I think it's fine for a short switch.